java/surface-placement/README.md

```diff
@@ -32,11 +33,11 @@ The application loads a scene showing three points that use the individual surfa
 
 ## About the data
 
-The scene launches with a view of northern Snowdonia National Park. Three points are shown hovering with positions defined by each of the different surface placement modes.
+The scene launches with a view of Brest, France. Four points are shown hovering with positions defined by each of the different surface placement modes.
 
 ## Additional information
 
-This sample uses an elevation service to add elevation/terrain to the scene. Graphics are positioned relative to that surface for the `DRAPED_BILLBOARDED`, `DRAPED_FLAT`, `ABSOLUTE` and `RELATIVE` surface placement modes. 
+This sample uses an elevation service to add elevation/terrain to the scene. Graphics are positioned relative to that surface for the `DrapedBillboarded`, `DrapedFlat`, and `Relative` surface placement modes. It also uses a scene layer containing 3D models of buildings. Graphics are positioned relative to that scene layer for the `RelativeToScene` surface placement mode.
```

Comment:
Should these still be all caps like the list above?

Reply:
good catch